src/ServiceFabric/ServiceFabric/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added support for Windows 2022 server vm image. 
+    - This enables cluster operations with Windows 2022 server vm image using AZ powershell. 
```

Comment:
Please pay attention to camel case by replacing `AZ powershell` with `Az PowerShell`. 
BTW as there is Azure PowerShell Changelog, it is not necessary to add `using AZ PowerShell`. Please remove it because in normal cases we simplify the Changelog sentences.

Reply:
@NoriZC , updated